_posts/2023-08-01-aitable.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "[CVE-2023-33665] Improper neutralization of a SQL parameter in aitable from ai-dev module for PrestaShop"
+categories: modules
+author:
+- 202-ecommerce.com
+- TouchWeb.fr
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,aitable"
+severity: "critical (9.8)"
+---
+
+In the module "Attributes table" (aitable) for PrestaShop, an attacker can perform a SQL injection up to 0.2.1. Release 0.2.2 fixed this security issue.
+
+## Summary
+
+* **CVE ID**: [CVE-2023-33665](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-33665)
+* **Published at**: 2023-08-01
+* **Advisory source**: Friends-Of-Presta.org
+* **Platform**: PrestaShop
+* **Product**: aitable
+* **Impacted release**: <= 0.2.1 (0.2.2 fixed issue)
+* **Product author**: ai-dev
+* **Weakness**: [CWE-89](https://cwe.mitre.org/data/definitions/89.html)
+* **Severity**: critical (9.8)
+
+## Description
+
+Up to 0.2.2, a sensitive SQL call in file `includes/ajax.php` can be executed with a trivial http call and exploited to forge a blind SQL injection throught the POST or GET submitted `attributes` variables.
+
+## CVSS base metrics
+
+* **Attack vector**: network
+* **Attack complexity**: low
+* **Privilege required**: none
+* **User interaction**: none
+* **Scope**: unchanged
+* **Confidentiality**: high
+* **Integrity**: high
+* **Availability**: high
+
+**Vector string**: [CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H)
+
+
+## Possible malicious usage
+
+* Technical and personal data leaks
+* Obtain admin access
+* Remove all data of the linked PrestaShop
+* Display sensitives tables to front-office to unlock potential admin's ajax scripts of modules protected by token on the ecosystem
```

Comment:
Display sensitive tables to the front office to unlock potential admins ajax scripts of modules protected by tokens on the ecosystem